docs/63_dandi_infrastructure.md

```diff
@@ -0,0 +1,185 @@
+# Work In Progress
+
+## Configuring Terraform
```

Comment:
@waxlamp @jjnesbitt @mvandenburgh would be great if you could review

For reference to easily read in staging setting: https://aquamarine-profiterole-e20e84.netlify.app/63_dandi_infrastructure/